app/views/bookings/_form.html.erb

```diff
@@ -4,12 +4,21 @@
   <%= f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present? %>
 
   <div class="form-inputs">
-    <%= f.hidden_field :client_id, value: current_user.id %>
+    <% if current_user.businesses.exists?(id: @booking.business_id) %>
+      <%= f.input :client_id, 
+                  collection: User.all.map { |u| [u.full_name, u.id] }, 
+                  label: "Select Client", 
+                  include_blank: "Choose a client" %>
+      <%= f.input :service_id, 
+                  collection: @business.services.map { |s| [s.name, s.id] }, 
+                  label: "Select Service", 
```

Comment:
nice and concise way to sign someone up for a service! hiding the client_id and service_id is a great way of filling out parts of the form!